docs/Developers/Writing_Good_Commits.rst

```diff
@@ -115,7 +115,7 @@ Example:
 
 ::
 
-    Fixes https://github.com/coala/coala/issues/269
+    Closes https://github.com/coala/coala/issues/269
```

Comment:
The newcomer may explore https://github.com/coala/coala/issues/269 to see that it is a bug. So should we change the link to an issue that should actually be addressed by `Closes` ? @adtac @dracarys09